gradle/tasks/clean.gradle

```diff
@@ -1,29 +1,39 @@
 def props = new Properties()
 
 task loadBuildFile() {
-    println "Load the build.properties file"
-    def buildPropsFile = file('build.properties')
-    if (buildPropsFile.exists()) {
-        buildPropsFile.withInputStream { props.load(it) }
-    } else {
-        println("WARNING: build.properties not found")
+    doLast {
+        println "Load the build.properties file"
+        def buildPropsFile = file('build.properties')
+        if (buildPropsFile.exists()) {
+            buildPropsFile.withInputStream {props.load(it)}
+        } else {
+            println("WARNING: build.properties not found")
+        }
     }
 }
 
-task cleanShared(type:Delete) {
-   description 'Removes ALL shared libraries from the servlet container.'
-   def serverbase = props.getProperty("server.home")
-   def libs =  "${serverbase}/shared/lib"
-   printf "Configured shared libraries from servlet container at location:  ${libs}"
-   FileTree tree = fileTree(dir: libs)
-   delete fileTree(dir:libs , include: '*')
+task cleanShared(type: Delete, dependsOn: 'loadBuildFile') {
+    group 'Clean Tomcat'
+    description 'Removes ALL shared libraries from the servlet container.'
+    followSymlinks = true
+
+    doFirst {
```

Comment:
The delete string needs to be set before the core work, deleting the files/directories, is performed. doLast{} does not set this variable in time.